### PR TITLE
DOC: Add code of conduct (issue #198 -- adopt code of conduct)

### PR DESCRIPTION
**Pull request by N-Dekker**
Adapted from https://www.contributor-covenant.org/version/1/4/code-of-conduct

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at M.Staring "AT" lumc.nl or
```
Comment (mstaring) [inline CODE_OF_CONDUCT.md]: use and instead of or
so they email us both

Comment (mstaring) [reply CODE_OF_CONDUCT.md]: but looks great otherwise!

**Comment by mstaring** [comment]
approved

**Comment by thewtex** [comment]
Nice!

**Pull request merged**